### PR TITLE
cleanup ether units page

### PR DESCRIPTION
**Pull request by pipermerriam**
This adds content and does some formatting cleanup to the ether page of the docs.

the-world s-top-10-best-images-of-cats-in-tiny-hats-9

### CODE REVIEW

File: source/using-ethereum-the-basics/ether.rst [hunk 1]
```diff
@@ -5,27 +5,58 @@ Ether
...
+| **Mwei (lovelace)** | 1e6 wei   | 1,000,000                                 |
+| **Gwei (shannon)**  | 1e9 wei   | 1,000,000,000                             |
+| **szabo**           | 1e12 wei  | 1,000,000,000,000                         |
+| **finney**          | 1e15 wei  | 1,000,000,000,000,000                     |
+| **ether**           | 1e18 wei  | 1,000,000,000,000,000,000                 |
+| **kether**          | 1e21 wei  | 1,000,000,000,000,000,000,000             |
+| **Mether**          | 1e24 wei  | 1,000,000,000,000,000,000,000,000         |
+| **Gether**          | 1e27 wei  | 1,000,000,000,000,000,000,000,000,000     |
+| **Tether**          | 1e30 wei  | 1,000,000,000,000,000,000,000,000,000,000 |
++---------------------+-----------+-------------------------------------------+
```
Comment (pipermerriam) [inline source/using-ethereum-the-basics/ether.rst]: This will render as a table in sphinx

File: source/using-ethereum-the-basics/ether.rst [hunk 2]
```diff
@@ -5,27 +5,58 @@ Ether
...
 
 Getting Ether
 ================================================================================
 
+Version 0.3.9 (Beta 6) of the `Ethereum Wallet`_ introduced the ability to
+purchase ether via Shape Shift.
+
+You can also purchase Ether on any exchange which supports the currency.  As of
+2015-02-12 Poloniex was the exchange with the highest volume in Ethereum
+transactions.
```
Comment (pipermerriam) [inline source/using-ethereum-the-basics/ether.rst]: Is this accurate?

File: source/using-ethereum-the-basics/ether.rst [hunk 3]
```diff
@@ -5,27 +5,58 @@ Ether
 What is Ether?
 ================================================================================
 
+Ether is the base unit of currency used within Ethereum.  It is used to pay for
```
Comment (5chdn) [inline source/using-ethereum-the-basics/ether.rst]: Isn't Wei the base unit? Ether is also often referred to as _crypto fuel_ required to run the blockchain.

File: source/using-ethereum-the-basics/ether.rst (hunk 2 above)
Comment (5chdn) [inline source/using-ethereum-the-basics/ether.rst]: Well, the version string belongs to Mist, the Ethereum browser. The DApp wallet has [no version tags yet](https://github.com/ethereum/meteor-dapp-wallet/tags). There is a lot of confusion in different places about this. Not sure how to deal with that or what the plans for Homestead actually are.

File: source/using-ethereum-the-basics/ether.rst (hunk 3 above)
Comment (Souptacular) [inline source/using-ethereum-the-basics/ether.rst]: Good point. I think we should refer to it as the crypto fuel required to run the blockchain or something similar.

**Pull request merged**